.gitignore

```diff
@@ -77,4 +77,532 @@ doc/
 # =========================================== #
 # Asset Script Reference Retargeter Generated #
 # =========================================== #
-NuGet/
\ No newline at end of file
+NuGet/
+/Assets.meta
```

Comment:
Did you mean to add all these to the .gitignore?

Reply:
No. Somehow these got added automatically 🙄 What's the best approach for reverting this? Is it from my side or can we cherry pick it somehow?

Reply:
I think I can revert. One sec!

Reply:
Done!

Reply:
Oh awesome! Thanks, Kurtis!! 😍